### DISCUSSION TITLE
How do I change the color of the text

### DISCUSSION THREAD

**Discussion by decide-you** [prasanthrangan/hyprdots#1860] [Q&A] [answered]
I'm sure this topic is covered in WIKI, but I'm too blind. How can I change the color of this dark text? Theme: Decay Green
![image](https://github.com/user-attachments/assets/6123c148-4789-4bbb-980a-cdd4d390327f)
![image](https://github.com/user-attachments/assets/f141ac9e-bbaf-4024-b2fe-8c86da4690eb)

**Comment by rubiin** [comment]
You mean the suggestion you are getting after sudo ? @decide-you 
If that is what you are asking,  
add this on your zzshrc file
`ZSH_AUTOSUGGEST_HIGHLIGHT_STYLE="fg=#50493e"`
change the color value to what you like

**Reply by rubiin** [reply]
more on that here https://github.com/zsh-users/zsh-autosuggestions#suggestion-highlight-style

**Reply by decide-you** [reply]
> You mean the suggestion you are getting after sudo ? @decide-you If that is what you are asking, add this on your zzshrc file `ZSH_AUTOSUGGEST_HIGHLIGHT_STYLE="fg=#50493e"` change the color value to what you like

thats it, thanks! and in 1st screenshot of dophin, the path to the open folder is too dark, you can see it on the 1st screenshot above. Also this dark text on a dark background appears during operations of merge folders. It's hard to see what it says

**Reply by rubiin** [reply]
could you circle the texts so I will get a clear idea

**Reply by decide-you** [reply]
> could you circle the texts so I will get a clear idea

![image](https://github.com/user-attachments/assets/77dd6d80-94ad-4826-ae56-d7fd89ef8fa2)

**Comment by decide-you** [comment]
@rubiin
to make it clear. in wiki "wallbash" section video preview theres a white path in the same theme (Tokyo Night), when mine are black (sorry for img quality)
Author's path text color:
![image](https://github.com/user-attachments/assets/b041cbea-966e-43c6-8f8b-d8f77555bcf7)
Mine:
![image](https://github.com/user-attachments/assets/738e2f71-4bdf-4b48-9788-f377aafae413)

**Comment by decide-you** [comment, marked as answer]
alright, i fixed this issue with updating my dotfiles to the latest....
